include/regs.h

```diff
@@ -277,6 +277,12 @@ typedef enum {
 #define R_PLAY_UCODE_DISAS_LOG_LEVEL HREG(92) // log level to use when switching to ucode disas
 
 // HREG_MODE_PERSPECTIVE
+#if OOT_DEBUG
+#define PERSPECTIVE_EDITOR_ACTIVE (R_HREG_MODE == HREG_MODE_PERSPECTIVE)
+#else
+#define PERSPECTIVE_EDITOR_ACTIVE false
+#endif
```

Comment:
I think we've started to prefer inlining `OOT_DEBUG && R_HREG_MODE == HREG_MODE_PERSPECTIVE` to this now (with the assumption that this will later be replaced by a macro